packages/test-app/config/ember-try.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
+
+module.exports = async function () {
+  return {
+    useYarn: true,
+    scenarios: [
+      {
+        name: 'ember-lts-3.24',
+        npm: {
+          devDependencies: {
+            'ember-source': '~3.24.3',
+            'ember-auto-import': '^2.2.0',
+          },
+        },
+      },
+      {
+        name: 'ember-lts-3.28',
+        npm: {
+          devDependencies: {
+            'ember-source': '~3.28.0',
+            'ember-auto-import': '^2.2.0',
+          },
+        },
+      },
+      {
+        name: 'ember-release',
+        npm: {
+          devDependencies: {
+            'ember-source': await getChannelURL('release'),
+          },
+        },
+      },
+      {
+        name: 'ember-beta',
+        npm: {
+          devDependencies: {
+            'ember-source': await getChannelURL('beta'),
+          },
+        },
+      },
+      {
+        name: 'ember-canary',
+        npm: {
+          devDependencies: {
+            'ember-source': await getChannelURL('canary'),
+          },
+        },
+      },
+      {
+        name: 'ember-default-with-jquery',
```

Comment:
@ctjhoa this one could be removed as it's not used and broken in Ember v4

Reply:
This is to match Ember 3.28 upgrade ( https://github.com/jmurphyau/ember-truth-helpers/pull/147/commits/c639111229e3107ddd0a82efa622067a4db96576 ).
https://github.com/ember-cli/ember-addon-output/blob/v3.28.5/config/ember-try.js#L50
Ember 4.0 is a step further to modernize the addon.